script/remove-zk-node.sh

```diff
@@ -37,11 +37,12 @@ source ${BIN_DIR}/env/dolphinscheduler_env.sh
 export JAVA_HOME=$JAVA_HOME
 
 export DOLPHINSCHEDULER_CONF_DIR=$DOLPHINSCHEDULER_HOME/conf
-export DOLPHINSCHEDULER_LIB_JARS=$DOLPHINSCHEDULER_HOME/lib/*
+export DOLPHINSCHEDULER_LIB_JARS=$DOLPHINSCHEDULER_HOME/libs/*
 
 export DOLPHINSCHEDULER_OPTS="-Xmx1g -Xms1g -Xss512k -XX:+DisableExplicitGC -XX:+UseConcMarkSweepGC -XX:+CMSParallelRemarkEnabled -XX:LargePageSizeInBytes=128m -XX:+UseFastAccessorMethods -XX:+UseCMSInitiatingOccupancyOnly -XX:CMSInitiatingOccupancyFraction=70 "
 export STOP_TIMEOUT=5
 
+registryServers=$REGISTRY_ZOOKEEPER_CONNECT_STRING
 CLASS=org.apache.zookeeper.ZooKeeperMain
 
 exec_command="$DOLPHINSCHEDULER_OPTS -classpath $DOLPHINSCHEDULER_CONF_DIR:$DOLPHINSCHEDULER_LIB_JARS $CLASS -server $registryServers rmr $rootNode"
```

Comment:
We should replace `$registryServers` to `$REGISTRY_ZOOKEEPER_CONNECT_STRING`. So that we can define one less variable.